src/main/java/org/mockito/internal/util/Platform.java

```diff
@@ -0,0 +1,48 @@
+package org.mockito.internal.util;
+
+public abstract class Platform {
+
+    public static final String JAVA_VERSION = System.getProperty("java.specification.version");
+    public static final String JVM_VERSION = System.getProperty("java.runtime.version");
```

Comment:
At the moment this shouldn't be a problem. the parsing targets Java 8 runtime

http://openjdk.java.net/jeps/223